### PR TITLE
feat: changed the style of dropdown properly

### LINKED ISSUE #523: [FEATURE] style dropdown properly
### Description

replace this:
https://github.com/Dun-sin/Whisper/blob/335b5cd9ce9ad49b0158ccbff3809a600824533e/client/src/components/Anonymous.jsx#L207-L221

with this: 
```
<Dropdown placement="leftStart" style={{ zIndex: 3 }} renderToggle={MenuToggle} noCaret>
  <Dropdown.Item onClick={() => handleClose()} className='sm:w-[200px]'>
    <div className="flex justify-between gap-2 flex-wrap">
      <span>Close Chat</span>
      <span className="text-gray-500 text-xs">Ctrl + Shift + X</span>
    </div>
  </Dropdown.Item>
  <Dropdown.Item onClick={() => handleClose(true)} className='sm:w-[200px]'>
    <div className="flex items-center justify-between gap-2 flex-wrap">
      <span>Find a new buddy</span>
      <span className="text-gray-500 text-xs">Ctrl + Alt + N</span>
    </div>
  </Dropdown.Item>
</Dropdown>
```

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by Roshanchau**
# Fixes #523

**My PR closes #issue_number_here**

# 👨‍💻 Changes proposed(What did you do ?)
- changed the style of the dropdown

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots

### CODE REVIEW

**Comment by Dun-sin** [comment]
> # PR Compliance Checks
> 
> Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project. The results are listed in the following section.
> ## Conventional Commit PR Title
> 
> In order to be considered for merging, the pull request title must match the specification in [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). You can edit the title in order for this check to pass. Most often, our PR titles are something like one of these:
> 
>     * docs: correct typo in README
> 
>     * feat: implement dark mode"
> 
>     * fix: correct remove button behavior
> 
> 
> Linting Errors
> 
>     * Found type "null", must be one of "feat","fix","docs","style","refactor","perf","test","build","ci","chore","revert"
> 
>     * No subject found
> 
> 
> ## Watched Files
> 
> This pull request modifies specific files that require careful review by the maintainers.
> 
> Files Matched
> 
>     * package-lock.json

@Roshanchau 
Please fix this error, if you don't know how, try this:
- Read the error carefully or read the contributing file
- Checkout past merged Pull requests to see how they were done for it to be merged

> Note: you can edit your PR message, this PR won't be merged if the error isn't gone and you have a week max to fix it before the PR gets closed without being merged

**Comment by Roshanchau** [comment]
I changed the commit message hope it is convenient to merge.

**Comment by Dun-sin** [comment]
> # PR Compliance Checks
> Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project.  The results are listed in the  following section.
> 
> ## Conventional Commit PR Title
> In order to be considered for merging, the pull request title must match the specification in  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). You can edit the title in order for this check to pass.
> Most often, our PR titles are something like one of these:
> - docs: correct typo in README
> - feat: implement dark mode"
> - fix: correct remove button behavior
> 
> Linting Errors
> 
> - Found type "null", must be one of "feat","fix","docs","style","refactor","perf","test","build","ci","chore","revert"
> - No subject found
> 
> ## Watched Files
> This pull request modifies specific files that require careful review by the maintainers.
> 
> Files Matched
> 
> - package-lock.json

@Roshanchau you are good to go when this error changes

**Comment by Roshanchau** [comment]
> > # PR Compliance Checks
> > Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project.  The results are listed in the  following section.
> > ## Conventional Commit PR Title
> > In order to be considered for merging, the pull request title must match the specification in  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). You can edit the title in order for this check to pass.
> > Most often, our PR titles are something like one of these:
> > 
> > * docs: correct typo in README
> > * feat: implement dark mode"
> > * fix: correct remove button behavior
> > 
> > Linting Errors
> > 
> > * Found type "null", must be one of "feat","fix","docs","style","refactor","perf","test","build","ci","chore","revert"
> > * No subject found
> > 
> > ## Watched Files
> > This pull request modifies specific files that require careful review by the maintainers.
> > Files Matched
> > 
> > * package-lock.json
> 
> @Roshanchau you are good to go when this error changes

**Comment by Roshanchau** [comment]
Is it done now ?

**Review by Dun-sin — CHANGES_REQUESTED** [review]
Remove all format changes and just leave the required change

**Comment by Roshanchau** [comment]
do you mean the format changes in the code or something else i didn't get it.

**Comment by Dun-sin** [comment]
> do you mean the format changes in the code or something else i didn't get it.

yes in the code

**Comment by Roshanchau** [comment]
I removed the format changes.

File: client/src/components/Anonymous.jsx [hunk 1]
```diff
@@ -262,20 +262,19 @@ const Anonymous = ({ onChatClosed }) => {
...
-
-					<Dropdown.Item onClick={() => handleClose(true)} className='sm:w-[200px]'>
-						<div className="flex items-center justify-between gap-2 flex-wrap">
-							<span>Find a new buddy</span>
-							<span className="text-gray-500 text-xs hidden sm:block">Ctrl + Alt + N</span>
-                        </div>
-                    </Dropdown.Item>
-				</Dropdown>
+  <Dropdown.Item onClick={() => handleClose()} className='sm:w-[200px]'>
+    <div className="flex justify-between gap-2 flex-wrap">
```
Comment (Dun-sin) [inline client/src/components/Anonymous.jsx:266]:
```suggestion
    <div className="flex items-center justify-between gap-2 flex-wrap">
```

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Pull request merged**